### ISSUE TITLE
[Bert/TensorFlow] Is auto loss scaling activated when setting tf.ConfigProto().graph_options.rewrite_options.auto_mixed_precision=1?

### ISSUE THREAD

**Issue by zhao1157** [NVIDIA/DeepLearningExamples#627]
In optimization.py, one way to use auto mixed precision is through

https://github.com/NVIDIA/DeepLearningExamples/blob/cd7d87074906f8e4299397b60c4142a13b88f9e6/TensorFlow/LanguageModeling/BERT/optimization.py#L99-L100

I wondered if I need an extra step to set the auto loss scaling `tf.train.experimental.DynamicLossScale(` when using 
`config = tf.ConfigProto()`
`config.graph_options.rewrite_options.auto_mixed_precision=1`

**Comment by swethmandava** [comment]
`tf.train.experimental.enable_mixed_precision_graph_rewrite` is equivalent to `config.graph_options.rewrite_options.auto_mixed_precision`. For a clear description, refer to [release notes](https://docs.nvidia.com/deeplearning/frameworks/tensorflow-release-notes/rel_19.09.html) here

**Comment by zhao1157** [comment]
Thanks.
